src/components/home/home.tsx

```diff
@@ -1,9 +1,20 @@
-// src/components/Home.tsx
 import React from 'react';
+import BrandLogo from '../../assets/svgs/brand-logo-magenta.svg';
+
+import './home.scss';
 
 const Home: React.FC = () => {
   return (
-    <div>
+    <div className="MagentaA11y--home-page">
+      <div className="MagentaA11y--home-page__header">
+        <div className="MagentaA11y--home-page__header--title">
+          <h1>Unlock the power of accessibility</h1>
+          <h2>MagentaA11y</h2>
+        </div>
+        <div className="MagentaA11y--home-page__header--logo">
+          <img src={BrandLogo} alt="Brand Logo" />
```

Comment:
depending on the image ultimately used, we should have more descriptive alt text such as alt="Globe surrounding T-Mobile T"